### ISSUE TITLE
Response Tab

### ISSUE THREAD

**Issue by alairock** [itsgoingd/clockwork#127]
Right now there is a ton of information you can get from the requests. All of the information needed really, but I was surprised the most basic of features didn't exist. Responses! Especially as I am working with API's, being able to see the actual response would be especially helpful.

**Comment by itsgoingd** [comment]
Hey, when using dev tools you already have your request data in the Network tab. I find myself often switching between Network and Clockwork tabs, I'd like to have a "show in Network tab" contextual menu  in Clockwork to make this a little bit easier, but I'll have to do some research to see if that's possible.

**Comment by itsgoingd** [comment]
I've realized I never explained why this is a challenging feature to add.

We could collect the full response in the Clockwork metadata, this is pretty straight-forward to implement. The main concern here is the response size, as your app might return anything from 1K of json to 100M binary blob. Most of the time you don't even care to see the response, so it's potentially a huge increase of metadata size with no benefit.

Another option would be to temporarily collect the response client-side in the browser extension. I don't like this as it would introduce feature inconsistency between the web and browser versions. Also you wouldn't be able to find responses for older requests.

I guess the best idea so far would be to introduce the response collection as optional (default off) feature with option to specify the required response type and possibly size limit.

**Comment by weidmaster** [comment]
Yes it would work better as an optional feature. It would be useful in tricky debugs with ajax data and what not. Thanks for considering the idea

**Comment by stepanorda** [comment]
@itsgoingd There are at least one use case when you have SPA with SSR and requests to API often happen in backend so you can't actually see responses in browser network tab.
